presto-docs/src/main/sphinx/release/release-337.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Release 337 (25 Jun 2020)
+=========================
+
+.. Note:: This release fixes a potential security vulnerability when secure internal communication is enabled in a cluster. A malicious
+   attacker can take advantage of this vulnerability to escalate privileges to internal APIs. We encourage everyone to upgrade as soon
+   as possible.
+
+General Changes
+---------------
+
+* Fix incorrect results for inequality join involving ``NaN``. (:issue:`4120`)
+* Fix peak non-revocable memory metric in event listener. (:issue:`4096`)
+* Fix queued query JMX stats. (:issue:`4129`)
+* Fix rendering of types in the output of ``DESCRIBE INPUT``. (:issue:`4023`)
+* Improve performance of queries involving comparisons between ``DOUBLE`` or ``REAL`` values and integer values. (:issue:`3533`)
+* Reduce idle CPU consumption in coordinator. (:issue:`3990`)
+* Add peak non-revocable memory metric to query stats. (:issue:`4096`)
+* Add support for variable-precision ``TIMESTAMP WITH TIME ZONE`` type (:issue:`3947`)
+* Add support for ``IN`` predicate with subqueries in outer join condition. (:issue:`4151`)
+* Add support for quantified comparisons (e.g., ``> ALL (...)``) in aggregation queries. (:issue:`4128`)
+* Add :doc:`/connector/druid`. (:issue:`3522`)
+* Add :func:`translate` function. (:issue:`4080`)
+* Reduce worker graceful shutdown duration. (:issue:`4192`)
+
+Security Changes
+----------------
+
+* Disable insecure authentication over HTTP by default when HTTPS with authentication is enabled. This
+  can be overridden via the ``http-server.authentication.allow-insecure-over-http`` configuration property. (:issue:`4199`)
+* Add support for insecure authentication over HTTPS to the Web UI. (:issue:`4199`)
+* Add :ref:`system_information_rules` which control the ability of a user to access to read and write system management information. (:issue:`4199`)
+* Disable user impersonation in default system security. (:issue:`4082`)
+
+Elasticsearch Connector Changes
+-------------------------------
+
+* Add support for password authentication. (:issue:`4165`)
+
+Hive Connector Changes
+----------------------
+
+* Fix reading CSV tables with ``separatorChar``, ``quoteChar`` or ``escapeChar`` table property
+  containing more than one character. For compatibility with Hive, only first character is considered
+  and remaining are ignored. (:issue:`3891`)
+* Improve performance of ``INSERT`` queries writing to bucketed tables when some buckets do not contain any data. (:issue:`1375`)
+* Improve performance of queries reading Parquet data with predicates on ``timestamp`` columns. (:issue:`4104`)
+* Improve performance for join queries over partitioned tables. (:issue:`4156`)
+* Add add support for ``null_format`` table property for tables using TextFile storage format (:issue:`4056`)
```

Comment:
Duplicate add here and below..

Reply:
Good catch! 

https://github.com/prestosql/presto/pull/4235